src/TaskRunner/Commands/InstallCommands.php

```diff
@@ -35,33 +31,20 @@ class InstallCommands extends AbstractCommands implements FilesystemAwareInterfa
    * - Check current status of configuration
    * - Import configuration from datastore into activestore.
    *
-   * @command toolkit:install-clone
+   * @command toolkit:install-clean-dev
    */
-  public function clone() {
-    // Check is dump is already download. @todo
-    if (!is_file('./.tmp/dump.sql')) {
-      if (is_file('./dump.sql')) {
-        if (!is_dir('.tmp')) {
-          $this->taskExec('mkdir -p .tmp')->run();
-        }
-        $this->taskExec('mv ./dump.sql  .tmp/')->run();
-      }
-      else {
-        // Get updated dump if the case.
-        $this->databaseDownload();
-      }
-    }
+  public function cleanDev() {
```

Comment:
Just for consistence with 3.x, please keep `toolkit:install-clean`, if the idea is to include developer tools here, we will create a task to enable developer tools in an installed site.

Reply:
changed.